src/librustc_span/symbol.rs

```diff
@@ -715,6 +715,7 @@ symbols! {
         target_has_atomic_load_store,
         target_thread_local,
         task,
+        _task_context,
```

Comment:
It's there because it suppresses the unused lint. I found that much easier than trying to create an `#[allow(unused)]` attribute in the HIR lowering code, but can do that if it's preferred.

Reply:
oh, I don't understand why the lint fires when you reference `sym::_task_context` elsewhere in the code. I was assuming you had it to suppress the lint at first, but didn't need it anymore.

Reply:
Oh, it doesn't fire on the compiler, if that's what you mean. It fires on the code being compiled when it doesn't `.await` anything IIRC.